iam-login-service/src/main/java/it/infn/mw/iam/api/aup/model/AupDTO.java

```diff
@@ -47,6 +48,11 @@ public class AupDTO {
   @Min(value = 0L, message = "Invalid AUP: signatureValidityInDays must be >= 0")
   Long signatureValidityInDays;
 
+  @NotNull(message = "Invalid AUP: aupRemindersInDays is required")
+  @Pattern(regexp = "\\d+,\\d+,\\d+",
```

Comment:
Spaces between commas and numbers should be accepted. Start and end of line should be added to the regexp.